### DISCUSSION TITLE
Older Bolt with Dshot ESCs

### DISCUSSION THREAD

**Discussion by AIUAV** [bitcraze/discussions#735] [Q&A] [answered]
Hi guys, as mentioned in the website of Bolt 1.1, Motor signal output M4 has been moved from PB9 to PB10 to be able to support the DSHOT protocol. 
1) I would like to know whether the older Bolt (before 1.1) can be modified to use DShot ESCs also? 
2) If not, how do we calibrate ESCs that use other calibration protocols, such as OneShot or MultiShot? 
Thank you!

**Comment by tobbeanton** [comment, marked as answer]
1. The Bolt 1.0 will be hard to run DSHOT on. Doing some patch to connect PB9->PB10 is the only option I could think of. Then the motor mapping would have to be adjusted as well since the Bolt 1.0 will still be identified as a 1.0

2. There is no "calibration" support other then altering code. Most often it will work perfectly fine without calibration though.
